misk/src/main/kotlin/misk/MiskServiceModule.kt

```diff
@@ -184,8 +182,21 @@ class MiskCommonServiceModule : KAbstractModule() {
 
 }
 
-internal data class DependencyEdge(val service: Key<*>, val dependency: Key<*>)
-internal data class EnhancementEdge(val service: Key<*>, val enhancement: Key<*>)
+/**
+ * Normal dependency edge in the service dependency graph.
+ *
+ * @param service Identifier for the dependent service.
+ * @param dependsOn Identifier for the service that [service] depends on.
+ */
+internal data class DependencyEdge(val service: Key<*>, val dependsOn: Key<*>)
```

Comment:
Be consistent! This should be `dependent`

Reply:
Done